ksy_schema.json

```diff
@@ -99,16 +99,23 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "id": { "$ref": "#/definitions/Identifier" },
-        "title": { "type": "string" },
+        "id": {
+          "$ref": "#/definitions/Identifier",
+          "description": "unique string that identifies this format. It should be identical to the file name without the `.ksy` extension, for example 'exe' for 'exe.ksy'. This name is used to infer name of the top-level type when generate the parser\n\nThis field is mandatory on top-level definition and shouldn't be used at nested levels"
```

Comment:
I think `'exe' for 'exe.ksy'` is a bad example, because AFAIK there are multiple formats that can be stored inside a file with the `.exe` extension (off the top of my head, at least http://formats.kaitai.io/dos_mz/ and http://formats.kaitai.io/microsoft_pe/). By giving the `exe.ksy` as an example, you may cause an impression that we advise naming the KSY file the same as the file extension - but that's not always a very good idea, as the 3-character abbreviation is rarely descriptive.

Reply:
Done